libparsec/crates/platform_sse/src/backoff.rs

```diff
@@ -0,0 +1,46 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use std::time::Duration;
+
+pub struct Limited {
```

Comment:
We should also provide some test for this (this should be pretty trivial)

Reply:
bb09fac84